.eslintrc.js

```diff
@@ -529,5 +529,16 @@ module.exports = {
         jquery: true,
       },
     },
+
+    /**
+     * TSVB overrides
+     */
+    {
+      files: ['src/legacy/core_plugins/metrics/**/*.js'],
+      rules: {
+        'import/no-default-export': 'error',
+      },
+      excludedFiles: 'src/legacy/core_plugins/metrics/index.js',
```

Comment:
Does this have the same effect as adding `!src/legacy/core_plugins/metrics/index.js` to the files list? I think that might be a little less surprising. If not, can you please move this next to the `files: []` list?

Reply:
@spalger, I've tried to write 
```
files: ['src/legacy/core_plugins/metrics/**/*.js', '!src/legacy/core_plugins/metrics/index.js'],
```
and it doesn't work as expected. Let me leave it as is. I don't see any concerns between `!path_to_file` and `excludedFiles ` approaches. 

P.S. The last one seems more obviously for me personally.